refarch-integrations/refarch-dms-integration/refarch-dms-integration-core/src/main/java/de/muenchen/refarch/integration/dms/adapter/out/fabasoft/FabasoftAdapter.java

```diff
@@ -68,6 +69,7 @@
 
 @Slf4j
 @RequiredArgsConstructor
+@SuppressWarnings({ "PMD.UseObjectForClearerAPI", "PMD.CouplingBetweenObjects" })
```

Comment:
Maybe we should just suppress the `PMD.UseObjectForClearerAPI` for now (and create a follow up issue to refactor all parts where this suppression is applied. Long term I agree with PMD it would be great to refactor the code to expose a clearer API.